status-im/status-react
# [ISSUE #2882] Updated wallet / send transaction to latest design

fixes #2882

### Summary:

Upgrade `wallet` `send` transaction flow to reflect latest mockups.
When possible, UI `components` have been introduced and reused where relevant.
Some global style changes have been applied (like uniformed border radius).

### Steps to test:
- Open Status
- Go to wallet
- Make sure `send` transaction flows are fine, both from `wallet` and `chat`

status: ready
